packages/http-client-java/generator/http-client-generator-test/src/test/java/azure/resourcemanager/nonresource/NonResourceTests.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azure.resourcemanager.nonresource;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.utils.ArmUtils;
+
+public final class NonResourceTests {
+
+    private final NonResourceManager manager
+        = NonResourceManager.authenticate(ArmUtils.createTestHttpPipeline(), ArmUtils.getAzureProfile());
+
+    @Test
+    public void testNonResourceGet() {
+        manager.nonResourceOperations().get("eastus", "hello");
+    }
+
+    @Disabled("It is mistakenly treated as a resource")
```

Comment:
The request can't be sent properly?

Reply:
The test require `id` property be sent with a value in PUT.

But if we take it as ARM resource, `id` cannot be set via `define` flow.

Reply:
I see.